backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/order/process/C_Order_CreatePOFromSOs.java

```diff
@@ -123,6 +129,17 @@ protected String doIt() throws Exception
 		}
 		workpackageAggregator.closeAllGroups();
 
+		final Set<OrderLineId> skippedSalesOrderLines = workpackageAggregator.getSkippedSalesOrderLineIds();
+
+		if (skippedSalesOrderLines.size() > 0)
+		{
+			final UserNotificationRequest userNotificationRequest = UserNotificationRequest.builder()
+					.recipientUserId(Env.getLoggedUserId())
+					.contentADMessage(MSG_SKIPPED_C_ORDERLINE_IDS)
```

Comment:
the C_OrderLine_ID don't tell the users anything. please output the `C_Order.DocumentNo` and `C_OrderLine.Line`.
`SkippedOrderLines`
- :speaking_head: EN `Skipped sales order lines {0}` => DE `Übersprungene Auftragszeilen: {0}`
- ...and the parameters could be a comma-delimited (`', '`) list of `<C_Order.DocumentNo>-<C_OrderLine.Line>`

Reply:
done